uint8 sid

bool ACTIVATE = True
bool DEACTIVATE = False

bool activate_state
